trickster/model.py

```diff
@@ -320,6 +415,15 @@ def get_response_validator_by_id(self, validator_id: uuid.UUID) -> ResponseValid
                 return validator
         return None
 
+    def authenticate(self, request: Request) -> None:
+        """Check if request is properly authenticated."""
+        self.auth.authenticate(request)
+
+    @field_serializer('auth')
+    def serialize_auth(self, auth: Auth, _info) -> dict[str, Any]:
```

Comment:
Is this method really needed, pydantic won't serialize Auth class by itself ?

Reply:
Even if it is in model which is returned from the API

router.post('/routes')
def create_route(route: InputRoute, mocked_router: Router = Depends(get_router)) -> Route:

....

return new_route


It is present in new route but without manual serialization it wont appear in response

Reply:
I think its because its returning Route which has "Auth" and in model its subclass like "ApiKeyAuth".. it ignores extra fields because they are not required by "Auth" and this was only way I could force it to dump extra fields